tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t4.py

```diff
@@ -150,8 +150,8 @@
                 ),
                 Field(
                     item="14",
-                    name="BLANK",
-                    friendly_name="blank",
+                    name="FAMILY_AFFILIATION",
```

Comment:
@elipe17 @reitermb @victoriaatraft i just realized something. Family affiliation is not a field in the T4 records. Please note:

- T4 record layout for tribal tanf is exactly the same as the T4 layout for tanf, which is located [here (see `TANF_SECT2_FAMILY` worksheet)](https://www.acf.hhs.gov/sites/default/files/documents/ofa/ACF-199-%26amp%3B-209-TANF-SSP-data-report-layouts-thru-2026-10.xlsx)
- item 14 is in the t5 record type and is already captured in tribal_tanf/t5.py
- my guess is that this blank field was being used to capture the trailing spaces in the T4 record. if true, i'd recommend changing the item number to mitigate confusion. We should also revert these changes back to `BLANK`